client/src/components/InviteIn.js

```diff
@@ -1,8 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+import CheckIcon from '@material-ui/icons/Check';
+import ClearIcon from '@material-ui/icons/Clear';
+import { makeStyles } from '@material-ui/core/styles';
 
-const InviteIn = (props) => {
-  return <div>InviteIn</div>;
+const useStyles = makeStyles((theme) => ({
+  inviteItem: {
+    height: 100,
+    width: '85%',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    borderBottom: 'solid 1px #ddd',
+    '& div': {
+      display: 'flex',
+      alignItems: 'center',
+    },
+  },
+  sideBarImg: {
+    borderRadius: '100%',
+    overflow: 'hidden',
+    width: '40px',
+    marginRight: theme.spacing(1),
+  },
+  button: {
+    background: 'none',
+    outline: 'none',
+    border: 'none',
+    cursor: 'pointer',
+  },
+  icon: {
+    fontSize: 30,
+  },
+  check: {
+    color: theme.palette.primary.blue,
+    '&:hover': {
+      color: theme.palette.secondary.green,
+    },
+  },
+  cross: {
+    '&:hover': {
+      color: 'red',
+    },
+  },
+}));
+
+const InviteIn = ({ invite, invitesIn, setInvitesIn }) => {
+  const { avatar, name } = invite;
+  const classes = useStyles();
+
+  const handleApprove = () => {
+    //remove invite from invitesIn
+    //change status to approved
+    //add to contacts
+
+    invite.status = 'approved';
+    let invitesInCopy = [...invitesIn];
+
+    let filteredInvites = invitesInCopy.filter((request) => {
+      return request.email !== invite.email;
+    });
+    console.log('invite:', invite);
+    setInvitesIn(filteredInvites);
+  };
+  const handleReject = () => {
+    //remove invite from invitesIn
+    invite.status = 'rejected';
+    let invitesInCopy = [...invitesIn];
+
+    let filteredInvites = invitesInCopy.filter((request) => {
+      return request.email !== invite.email;
+    });
+    console.log('invite:', invite);
+    setInvitesIn(filteredInvites);
```

Comment:
Same here. Also, add a TODO comment about the need to make an API call to update this in the database too